ffmpeg/ffmpeg.go

```diff
@@ -798,6 +861,12 @@ func InitFFmpegWithLogLevel(level LogLevel) {
 	C.lpms_init(C.enum_LPMSLogLevel(level))
 }
 
+func InitFFmpegWithXcoderParams(param string) {
+	fmt.Println("InitFFmpegWithXcoderParams: ", param)
+	ts_param := C.CString(param)
```

Comment:
`ts_param` not freed manually. Maybe do `deffer` here?

Reply:
removed as it is unused